### PR TITLE
resolve issue #1

### PR DESCRIPTION
**Pull request by wlabsalin**
Who can review?

### CODE REVIEW

**Comment by wlabpgliu** [comment]
I'm OK.

File: README.md
Comment (wlabpmlee) [inline README.md:1]: OK

**Review by wlabpmlee — COMMENTED** [review]
OK

**Review by wlabpmlee — APPROVED** [review]
Approve

**Comment by wlabpmlee** [comment]
Issue #1 solved.

**Comment by wlabpmlee** [comment]
done

**Pull request closed without merging**